### PR TITLE
feat(app-platform): Add columns to SentryApp and add SentryAppAvatar

### PR DESCRIPTION
**Pull request by MeredithAnya**
* Adds `redirect_url` column
* Adds `overview` column (for sentry apps marketplace page)
* Changes `webhook_url` to use `URLField`
* Adds `SentryAppAvatar` model (for sentry apps marketplace page)

### CODE REVIEW

File: src/sentry/models/sentryappavatar.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+from . import AvatarBase
+
+
+class SentryAppAvatar(AvatarBase):
+    """
+    A SentryAppAvatar associates a SentryApp with an avatar photo File
+    and contains the preferences for avatar type.
+    """
+
+    AVATAR_TYPES = ((0, 'letter_avatar'), (1, 'upload'), )
```
Comment (mnoble) [inline src/sentry/models/sentryappavatar.py]: Quibble: don't think you need the trailing `,` since you have two items.

**Comment by mnoble** [comment]
cc @mattrobenolt @JTCunning Again, there's no records in this table, so shouldn't affect anything.

File: src/sentry/models/sentryapp.py [hunk 2]
```diff
@@ -51,7 +51,10 @@ class SentryApp(ParanoidModel, HasApiScopes):
     uuid = models.CharField(max_length=64,
                             default=default_uuid)
 
-    webhook_url = models.TextField()
+    redirect_url = models.URLField()
```
Comment (mnoble) [inline src/sentry/models/sentryapp.py]: Let's make this one `null=True` for anyone who's service can function completely without User interaction. The `webhook_url` would be enough for that.

**Review by mnoble — CHANGES_REQUESTED** [review]

**Comment by mattrobenolt** [comment]
Yeah, all fine with me if there are no records.

**Review by mnoble — APPROVED** [review]
👍

**Pull request merged**